src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx

```diff
@@ -33,19 +35,23 @@ export function ImportTokenModal(props: ImportTokenModalProps) {
   )
 
   // dismiss warning if all imported tokens are in active lists
-  const defaultTokens = useAllTokens()
+  const tokensByAddress = useAtomValue(tokensByAddressAtom)
+  const tokensBySymbol = useAtomValue(tokensBySymbolAtom)
   // example: https://cowswap.dev.gnosisdev.com/#/swap?chain=mainnet&inputCurrency=0xe0b7927c4af23765cb51314a0e0521a9645f0e2a&outputCurrency=0x539F3615C1dBAfa0D008d87504667458acBd16Fa
   const importTokensNotInDefault = useMemo(() => {
     // We should return an empty array until the defaultTokens are loaded
     // Otherwise WETH will be in urlLoadedTokens but defaultTokens will be empty
     // Fix for https://github.com/cowprotocol/cowswap/issues/534
-    if (!Object.keys(defaultTokens).length) return []
+    if (!Object.keys(tokensByAddress).length) return []
 
     return (
       urlLoadedTokens &&
       urlLoadedTokens
         .filter((token: Token) => {
-          return !Boolean(token.address.toLowerCase() in defaultTokens)
+          return (
+            !Boolean(token.address.toLowerCase() in tokensByAddress) &&
+            !Boolean(token.symbol!.toLowerCase() in tokensBySymbol)
```

Comment:
What's the name of this operator and what exactly does it do? `!.`
Sorry for the newbie-ness, don't think I saw that before

Reply:
It's non-null operator: https://www.logicbig.com/tutorials/misc/typescript/non-null-assertion-operator.html

`Token` interface has `symbol` as an optional parameter. But it's not possible for us.

Reply:
Ah ok, thank you!

Why not use `?.`?
If I remember correctly using that operator was a linter/compiler warning or error

Reply:
Yes, you are right. I changed the commit in the hotfix branch